src/DataGrid.tsx

```diff
@@ -842,7 +842,7 @@ function DataGrid<R, SR>({
           onRowClick={onRowClick}
           rowClass={rowClass}
           top={top}
-          copiedCellIdx={copiedCell?.row === row ? columns.findIndex(c => c.key === copiedCell.columnKey) : undefined}
+          copiedCellIdx={copiedCell !== null && copiedCell.row === row ? columns.findIndex(c => c.key === copiedCell.columnKey) : undefined}
```

Comment:
We are looping through rows and rows type is `readonly R[];`. I am not sure when can row be undefined?

Reply:
`R` can be `undefined` like in our million cells demo.